.github/renovate.json

```diff
@@ -1,7 +1,68 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "enabledManagers": ["github-actions"],
+  "enabledManagers": ["github-actions", "npm"],
+  "ignorePaths": ["packages/engine/**"],
   "prConcurrentLimit": 5,
   "rebaseWhen": "conflicted",
-  "schedule": ["every weekday before 5am"]
+  "schedule": ["every weekday before 5am"],
+  "packageRules": [
+    {
+      "automerge": true,
```

Comment:
Experiment: https://docs.renovatebot.com/configuration-options/#automerge

According to the docs, `automerge` + `stabilityDays` works the following way:
- A PR is created right after a new version is out
- Renovate status check remains pending till `stabilityDays` is reached
- If all status checks are green, the PR is merged automatically.

In our case, all PRs need to be reviewed by humans. This means that Renovate won't merge anything by itself. However, we'll get PRs for new versions immedately instead of after `stabilityDays`.

Reply:
Yeah I agree. What's good about `automerge` is that it creates PRs right away, then waits. With `stabilityDays` being set in BP without `automerge`, we can see a PR that mentions a dependency that's no longer latest.

Reply:
Cool, sounds good